tests/nd_item/nd_item_async_work_group_copy_fp16.cpp

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+//
+//  SYCL 1.2.1 Conformance Test Suite
+//
+//  Copyright:	(c) 2017 by Codeplay Software LTD. All Rights Reserved.
+//
+*******************************************************************************/
+
+#include "nd_item_async_work_group_copy_common.h"
+
+#define TEST_NAME nd_item_async_work_group_copy_fp16
+
+namespace TEST_NAMESPACE {
+
+class TEST_NAME : public util::test_base {
+ public:
+  /** return information about this test
+  */
+  void get_info(test_base::info &out) const override {
+    set_test_info(out, TOSTRING(TEST_NAME), TEST_FILE);
+  }
+
+  /** execute the test
+  */
+  void run(util::logger &log) override {
+
+    if(makeQueueOnce().get_device().has_extension("cl_khr_fp16")){
```

Comment:
Please, log about skipping the tests in all *[fp16,fp64].cpp tests.
Like this https://github.com/KhronosGroup/SYCL-CTS/blob/SYCL-1.2.1/master/tests/accessor/accessor_api_buffer_fp16.cpp#L37.